### PR TITLE
Feature/admin settings

### PR DESCRIPTION
**Pull request by kkuepper**

### CODE REVIEW

File: frontend/pages/upload/bmm.vue [hunk 1]
```diff
@@ -1,64 +1,112 @@
...
 
+const selectedEnvironment = ref(BmmEnvironment.Production);
+watch(form, (newForm) => {
+    const newEnvironment =
+        newForm.environment === "int"
+            ? BmmEnvironment.Integration
+            : BmmEnvironment.Production;
+    if (selectedEnvironment.value !== newEnvironment)
+        selectedEnvironment.value = newEnvironment;
+});
```
Comment (andreasgangso) [inline frontend/pages/upload/bmm.vue:27]: a `computed` would be a better fit here
`const selectedEnvironment = computed(() => form.environment === "int" ? BmmEnvironment.Integration : BmmEnvironment.Production)`

File: frontend/components/SelectFile.vue [hunk 2]
```diff
@@ -15,13 +16,14 @@ const dragLeave = () => {
...
     isDragOver.value = false;
     const files = event.dataTransfer?.files;
-    selectedFile.value = files?.[0] ?? null;
+    const file = files?.[0];
+    if (file) selectedFiles.value.push({ file, language: "nb" });
 };
 
 const fileInput = ref<HTMLInputElement>(null!);
 
 const selectFile = (event: any) => {
```
Comment (andreasgangso) [inline frontend/components/SelectFile.vue:25]: `: any` 😢

Comment (andreasgangso) [reply frontend/components/SelectFile.vue:25]: `event: Event & { target: EventTarget & HTMLInputElement }`

File: frontend/components/LanguageSelector.vue [hunk 3]
```diff
@@ -12,15 +12,19 @@ const bmmLanguages = ref<string[]>([]);
...
-}, {immediate: true});
-
+watch(
+    [() => props.env, () => props.languages],
+    async ([newEnv, newLanguages]) => {
+        if (newLanguages.length > 0) {
+            bmmLanguages.value = newLanguages;
+        } else {
+            bmmLanguages.value = (
+                await api.getLanguages({ environment: newEnv })
```
Comment (andreasgangso) [inline frontend/components/LanguageSelector.vue:22]: really dont like the formatting. i tried to tweak prettier but couldnt figure it out.
not important for the pr

**Review by andreasgangso — APPROVED** [review]
lgtm just minor things

File: frontend/pages/upload/bmm.vue (hunk 1 above)
Comment (kkuepper) [reply frontend/pages/upload/bmm.vue:23]: done

File: frontend/components/SelectFile.vue (hunk 2 above)
Comment (kkuepper) [reply frontend/components/SelectFile.vue]: done

**Pull request merged**